src/Image/Image.php

```diff
@@ -24,6 +28,7 @@ final class Image
     public function __construct(array $data)
     {
         $this->id = $data['id'];
+        $this->url = $data['url'];
```

Comment:
```suggestion
        $this->url = array_map(function(string $url): UriInterface {
               return new Uri($url);
        }, \is_array($data['url']) ?? []);
```
Ca permet de savoir ce que contient ton tableau